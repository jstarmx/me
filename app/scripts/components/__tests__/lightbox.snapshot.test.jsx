const Lightbox = require('../lightbox');
const React = require('react');
const renderer = require('react-test-renderer');

describe('<Lightbox /> (Snapshot)', () => {
  it('renders a lightbox container', () => {
    const component = renderer.create(<Lightbox />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
