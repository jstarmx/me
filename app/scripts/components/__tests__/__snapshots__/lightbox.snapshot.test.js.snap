exports[`Lightbox (Snapshot) renders a lightbox container 1`] = `null`;
